.github/workflows/ci.yaml

```diff
@@ -119,5 +125,98 @@ jobs:
         working-directory: ${{ matrix.build-dir }}
         run: cargo test
 
+  "vscode_extension_format":
```

Comment:
This should probably be a separate workflow since there is no dependency between the two.

We should also make it so that it only runs when the extension changes I think. It looks like it will be somewhat expensive to run.

Vice-versa we should only run cargo-concordium one when that folder or base dependency changes.